README.md

```diff
@@ -110,8 +110,6 @@ With git bash,
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
 - [ ] **Javascript**
 - [ ] **React**
 
```

Comment:
- [ **OPTIONAL** ] Kindly ensure that all your project's contributors are present in your `# Authors` section of your README to make your documentation more professional